.gitignore

```diff
@@ -2,7 +2,11 @@
 .Rhistory
 .RData
 .Ruserdata
+**/*.Rproj
+**/renv/
```

Comment:
Appreciate this advice--I definitely thrashed around trying to piece together the renv pieces that would work universally. If I understand correctly, making this revision work would require putting an .Rproj file at the root of the repo (if that's wrong, ignore the rest of this comment 😅 )

I wanted to contain the R and Python-related files in the `src/` directory and keep the rest of the repo as "pure hub" because it struck me as tidier. I see that the tradeoff is some extra steps for setup.

I'll file the advice away for next time and leave this alone for now because the the script is intended to be run as part of a scheduled GitHub action and not a human (and because it took me a while to get the action working). Happy to revisit in the future if needed!